lib/backup_helper.py

```diff
@@ -0,0 +1,44 @@
+import tarfile
+from os import path, getenv, remove
+from shutil import rmtree
+import sh
+
+directories = ['.screenly', 'screenly_assets']
+archive_name = "screenly-backup.tar.gz"
+
+
+def create_backup():
+    home = getenv('HOME')
+    file_path = path.join('static', archive_name)
+    if path.isfile(file_path):
+        remove(file_path)
+
+    try:
+        with tarfile.open(file_path, "w:gz") as tar:
+
+            for directory in directories:
+                path_to_dir = path.join(home, directory)
+                tar.add(path_to_dir, arcname=directory)
+    except IOError as e:
+        remove(file_path)
+        raise e
+
+    return archive_name
+
+
+def recover(file_path):
+    home = getenv('HOME')
+
+    with tarfile.open(file_path, "r:gz") as tar:
+        for directory in directories:
+            if directory not in tar.getnames():
+                raise Exception("Archive is wrong.")
+
+        sh.sudo('service', 'screenly-viewer', 'stop')
+        for directory in directories:
+            rmtree(path.join(home, directory))
+
+        tar.extractall(path=path.join(home))
+        sh.sudo('service', 'screenly-viewer', 'start')
```

Comment:
We can't do this as we cannot assume there is no password set. Instead, we need to do something like this:

 * Create a new file in Ansible called something like this `/usr/local/bin/screenly_util.sh`
 * Use argparse or similar to make allow it to take a variable like `screenly_util.sh backup /path/to/backup.tgz`
 * Configure `/etc/sudoers` to allow the user `pi` to execute this command with `sudo` passwordless